telethon/client/auth.py

```diff
@@ -153,13 +153,13 @@ async def _start(
                     warnings.warn(
                         'the session already had an authorized user so it did '
                         'not login to the bot account using the provided '
-                        'bot_token (it may not be using the user you expect)'
+                        f'bot_token (it is using ID {me.id})'
                     )
             elif phone and not callable(phone) and utils.parse_phone(phone) != me.phone:
                 warnings.warn(
                     'the session already had an authorized user so it did '
                     'not login to the user account using the provided '
-                    'phone (it may not be using the user you expect)'
+                    f'phone (the account is using phone number {me.phone})'
```

Comment:
I don't generally like logging personal information, would this be fine with user ID again?

Reply:
I agree, but it doesn't really make sense to log the ID when the phone is wrong, and it could make it even more confusing. Perhaps better just to not do anything in this hunk.

Reply:
I'm not sure about the changes as a whole, this is just meant to be a warning that "hey you probably didn't mean to do that" and the user should probably not do that.